src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -83,13 +87,5 @@ $spacing: 20px;
   &__events {
     max-height: 35vh;
     overflow-y: auto;
-    // @debt these overrides are here because ScheduleItem sets incompatible margins/etc.
-    //   We should just clean it up there instead.
-    .schedule-item-container {
-      margin: -10px;
-      padding: 10px $spacing;
-
-      border-radius: $border-radius;
-    }
```

Comment:
`<ScheduleItem/>` component is used only for `RoomModal`. There is no need to override its styles